node-go/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:1.15.7
 RUN wget -O /usr/local/bin/nancy "https://github.com/sonatype-nexus-community/nancy/releases/download/v1.0.11/nancy-v1.0.11-linux-amd64" \
     && chmod 755 /usr/local/bin/nancy
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash
+RUN curl -sL https://deb.nodesource.com/setup_13.x | bash
 
 RUN apt-get update && \
     apt-get -y upgrade && \
```

Comment:
It looks like the line below this still references version 8 and needs to be updated.